### DISCUSSION TITLE
Calling composite action inside container for execution.

### DISCUSSION THREAD

**Discussion by kamilhitachi** [community/community#34441] [Actions]
How can we call composite action inside container for execution : 
 - name: Calling build Composite Action
        uses: ./.github/composite/build/
        with:
           image: maven:3.8.6-jdk-11

will above composite action run inside the image mentioned in with:

**Comment by airtower-luna** [comment]
> will above composite action run inside the image mentioned in with:

No, it won't. Most likely you should look into using a [Docker action](https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action) instead, or for the parts of the composite action that'd need it.

Another option may be to include `docker run` commands in the composite action (possibly with the image name coming from an input), but getting the volume mounts right may be tricky.
